crates/tuono/Cargo.toml

```diff
@@ -21,6 +21,7 @@ path = "src/lib.rs"
 
 [dependencies]
 clap = { version = "4.5.4", features = ["derive", "cargo"] }
+tempfile = "3.14.0"
```

Comment:
`tempfile` should be added to the [dev-dependencies](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#development-dependencies)